pyvista/core/filters/data_set.py

```diff
@@ -511,17 +511,30 @@ def slice_along_axis(dataset, n=5, axis='x', tolerance=None,
             axis index (``0``, ``1``, or ``2``).
 
         tolerance : float, optional
-            The tolerance to the edge of the dataset bounds to create the
-            slices.
+            The tolerance to the edge of the dataset bounds to create
+            the slices. The ``n`` slices are placed equidistantly with
+            an absolute padding of ``tolerance`` inside each side of the
+            ``bounds``- along the specified axis. Defaults to 1% of the
```

Comment:
Is the `-` meant to be here?

Reply:
Absolutely not! Thanks for spotting it :)

Reply:
Fixed in https://github.com/pyvista/pyvista/pull/1506/commits/31dd0f2ef1ec1c32b9235880938fdb092915c05f